uint8 value

# defined enums that have to be used for status
uint8 OK=0
uint8 CHARGER_CONNECTED=1
uint8 MOTOR_ERROR=2
uint8 LOW_BATTERY=3
uint8 EMERGENCY_STOP=4
